tests/Form/SelectTest.php

```diff
@@ -63,7 +70,8 @@ public function testElementSelectedStateCheck($selectName, $optionValue, $option
         $session->visit($this->pathTo('/multiselect_form.html'));
         $select = $webAssert->fieldExists($selectName);
 
-        $option = $webAssert->elementExists('named', array('option', $optionValue));
+        $option = $select->find('named', array('option', $optionValue));
+        self::assertInstanceOf('Behat\Mink\Element\NodeElement', $option);
```

Comment:
Is this needed because the `$webAssert->elementExists` is no longer used?

Reply:
Yes. But it's not relevant anymore.